source/clear-linux/guides/network/ovs-dpdk.rst

```diff
@@ -187,111 +188,109 @@ Using OpenvSwitch
             -net nic,model=virtio,macaddr=00:11:22:33:44:55 -net tap,script=ovs-ifup,downscript=ovs-ifdown \
             -debugcon file:debug.log -global isa-debugcon.iobase=0x402
 
-#. Run guest virtual machine B using the configuration from step 5, only
-   it's necessary to change the MAC address to something like *00:11:22:33:44:56*
+#. Run guest virtual machine B using a similar reference configuration. Be
+   sure to change the MAC address to a different value.
 
-#. Follow the instructions in the `Setting IP address`_ section.
+#. Follow the instructions in the :ref:`set-ip-addr-ovs-dpdk` section.
 
-#. Alternatively, clean the previous environment, turn off the virtual
+#. When testing is complete, clean the previous environment, turn off the virtual
    machines, and delete the bridge.
 
    .. code-block:: bash
 
-      # ovs-vsctl del-br br0
-      # ovs-vsctl show
+      sudo ovs-vsctl del-br br0
+      sudo ovs-vsctl show
+
 
-Using Linux OpenvSwitch-DPDK
-============================
+Use custom-built Open vSwitch-DPDK bridge
+*****************************************
 
 #. Enable VT-d technology in the BIOS.
```

Comment:
Would it help users to add steps to enable VT-d in the BIOS? If that task is either extremely basic or too complex for this guide, please reject this comment.

Reply:
There are several different BIOS variants out there with different ways to make the change.  It is better to just generally tell a user what the need to do to enable VT-d in their BIOS

Reply:
Got it, thanks Tom.